packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -116,9 +117,19 @@ async function fetchDestinations(
 
   return callDestinationEndpoint(targetUri, headers, options)
     .then(response => {
-      const destinations: Destination[] = response.data.map(d =>
-        parseDestination(d)
-      );
+      const destinations: Destination[] = response.data
+        .filter(dest => {
+          try {
```

Comment:
Question: Why use try/catch for control flow here? My suggestion was to make the `validateDestinationConfig` have a bool return type and use that return value for the filter.

I'm open to learn about the advantages of using try/catch here, but to me this seems like using it for control flow which feels odd.

Reply:
I wanted to reuse the existing method, which throws and exception. If I would write the message new, I would use a bool to fit the filter format.

Reply:
In that case I would prefer adding a new method with that interface and having it internal, but that's not a hard requirement for me to approve.